src/cmd/nsfs.js

```diff
@@ -324,10 +325,38 @@ async function main(argv = minimist(process.argv.slice(2))) {
 
     } catch (err) {
         console.error('nsfs: exit on error', err.stack || err);
+        //noobaa crashed
+        dbg.event({
+            code: "noobaa_nsfs_crashed",
+            entity_type: "NODE",
+            event_type: "STATE_CHANGE",
+            message: "Noobaa NSFS crashed with error : " + err,
+            scope: "NODE",
+            severity: "ERROR",
+            state: "STOPPED"
+        });
         process.exit(2);
     }
 }
 
+async function verify_gpfslib(backend, fs_config) {
+    if (backend === 'GPFS' && nb_native().fs.gpfs) {
```

Comment:
I don't think we are really using backend === 'GPFS', I think you can remove this condition